openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/widgets/VWidget.js

```diff
@@ -200,7 +200,11 @@ export default {
       // Only convert the first 32 bytes before adding an ellipse
       // TODO: Handle units on a BLOCK item
       // TODO: Render data in a BLOCK item as bytes (instead of ASCII)
-      if (value['json_class'] === 'String' && value['raw'] !== undefined) {
+      if (
+        value &&
+        value['json_class'] === 'String' &&
+        value['raw'] !== undefined
+      ) {
```

Comment:
This is a critical bug fix that prevents and values from being displayed in TlmViewer screens. Introduced in my last binary display PR.